app/components/Account/FeePoolOperation.jsx

```diff
@@ -79,7 +79,7 @@ class FeePoolOperation extends React.Component {
         claimCollateralFeesAmount: 0,
         claimCollateralFeesAmountAsset: new Asset({
             amount: 0,
-            precision: this.props.asset.get("precision"),
+            precision: this.props.core.get("precision"),
```

Comment:
This looks wrong. In fact, the asset type of the collateral fee is just the backing asset (see code below), it could be another asset, but not only core.